backend/src/main/java/gov/cdc/usds/simplereport/service/dataloader/PatientPrimaryPhoneDataLoader.java

```diff
@@ -18,6 +19,8 @@ public String getKey() {
     return KEY;
   }
 
+  private static final int limit = 5000;
```

Comment:
Not requesting a change, but FYI this is something SonarCloud complains about - it wants constants in `SNAKE_CASE_ALL_CAPS`.

If you haven't noticed already, pushing any changes to an open PR wipes out any existing reviews so I would actually ask you _not_ to address this 😂 just something to file away in your memory bank